systems/plants/shapes/Geometry.cpp

```diff
@@ -35,6 +44,26 @@ namespace DrakeShapes
     return new Sphere(*this);
   }
 
+  void Sphere::getPoints(Matrix4Xd &points) {
+    points = Matrix4Xd::Zero(4,1);
+    points(3,0) = 1;
+  }
+
+  void Sphere::getBoundingBoxPoints(Matrix4Xd &points) {
+      // Return axis-aligned bounding-box vertices
+      points.resize(4,8);
```

Comment:
extract magic number 8 out into a constant (to be used on line 56 as well) and use Eigen::NoChange instead of 4.